src/components/Pages/Rocket/RocketList.js

```diff
@@ -0,0 +1,44 @@
+/*eslint-disable*/
+import "./Rocket.css"
+import FetchApi from "../../../Redux/rockets/Api";
+import { useEffect } from "react";
+import { useSelector, useDispatch } from "react-redux";
+import { toggleRocketStatus } from "../../../Redux/rockets/rocketsSlice";
+import styles from './rocket.module.css';
+
+export const RocketList = () => {
+    const { error, rocketStatus, rockets } = useSelector((state) => state.rocket);
+    const dispatch = useDispatch();
+    useEffect(() => {
+        dispatch(FetchApi());
+    }, [dispatch]);
+    console.log(FetchApi);
+    
```

Comment:
- [ ] As per JavaScript best practices, it is not advisable to commit `console.log` to your repo. For the reason that it’s ugly, it kills performance and it can make confidential data visible to anyone using the browser tools to look at your website.

Reply:
thanks for suggesting me 👍